.prettierignore

```diff
@@ -0,0 +1,3 @@
+# Because django templates requires syntax which can't contain line break
```

Comment:
In your commit message:

`s/s it recommended in python to respect line width of 79 characters,/It is recommened in Python to respect a line width of 79 characters,/`
`s/I propose to use the Airbnb standard/I suggest to use the Airbnb standard/`